GPCS4/SceModules/SceLibkernel/sce_libkernel.cpp

```diff
@@ -47,6 +49,40 @@ int PS4API __pthread_cxa_finalize(void)
 }
 
 
+int PS4API scek__write(int fd, const void* buf, size_t size)
+{
+	if (fd > 2)
+	{
+		LOG_FIXME("Not implemented for non-stdout");
+	}
+	else
+	{
+		FILE* outStdStream{};
+
+		switch (fd) {
+		case 1:
+			outStdStream = stdout;
+			break;
+		case 2:
+			outStdStream = stderr;
+			break;
+		default:
+			return SCE_OK;
+		}
+
+		// Log it to logger
+		LOG_SCE_TRACE("%s", buf);
```

Comment:
there need to write `'\0'` at `size` position (and probably remove eol), because buf not always null-teminated

Reply:
For strings (which is what we actually print), it will be null terminated.

Reply:
no, it don't. i checked it already, because of explicit size not always string zero-terminated